infrastructure/terraform/accounts/modules/account/aws/outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "admin_role_arn" {
+  value = "${module.admin_role.arn}"
```

Comment:
nit: in terraform 0.12, interpolating expressions like this is deprecated

drop the quotes and `${…}`.